src/main/java/org/gridsuite/actions/server/ContingencyListController.java

```diff
@@ -93,16 +93,17 @@ public ContingencyListController(ContingencyListService service) {
             .orElse(ResponseEntity.notFound().build());
     }
 
-    @PostMapping(value = "script-contingency-lists/", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @PostMapping(value = "/script-contingency-lists", consumes = MediaType.APPLICATION_JSON_VALUE)
     @Operation(summary = "Create a script contingency list")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The script contingency list have been created successfully")})
-    public ResponseEntity<ScriptContingencyList> createScriptContingencyList(@RequestBody(required = false) ScriptContingencyList script) {
+    public ResponseEntity<ScriptContingencyList> createScriptContingencyList(@RequestParam(required = false, value = "id") UUID id,
```

Comment:
Shouldn't the id be required and fixed by the directory-server when the creation request is received ?

Reply:
It is generated by the directory server, if we create the contingency in gridExplore
But, as gridActions app is still present, we must keep the id optional